get_session_request.php

```diff
@@ -0,0 +1,54 @@
+<?php
+require_once 'vendor/autoload.php';
+require_once 'config.php';
+
+use \Firebase\JWT\JWT;
+
+function get_session_request($contents)
+{
+    if (JWT_ENABLED) {
+        $jwt_pk = file_get_contents(IRMA_SERVER_PUBLICKEY);
+        try {
+            $decoded = JWT::decode($contents, $jwt_pk, array('RS256'));
+            $fullname = $decoded->disclosed[0][0]->rawvalue;
+        } catch (Exception $e) {
+            error_log("JWT could not be parsed: " . $e);
+            header("HTTP/1.0 403 Forbidden");
+            exit;
+        };
+    } else {
+        $parsedjson = json_decode($contents, true);
+        if (is_null($parsedjson)) {
+            error_log("JSON could not be parsed.");
+            header("HTTP/1.0 403 Forbidden");
+            exit;
```

Comment:
This branch, too, has essentially nothing to do with whether the IRMA server sends private keys or not.

If you pass [`false` as the second parameter to `json_decode`](https://www.php.net/manual/en/function.json-decode.php), it will return an object instead of an array, so that you can lookup the disclosed attribute in `$parsedjson` the same as you do in the JWT branch above. This will allow you to move this null check as well as looking up the disclosed attribute to outside the branch.